spectrum-network/src/peer_manager.rs

```diff
@@ -1,14 +1,17 @@
 pub mod data;
```

Comment:
this module contains lot of logic, you can divide it into several submodules (e.g PeersMailbox.rs and etc) to make it more meaningful